Gruntfile.js

```diff
@@ -42,7 +43,7 @@ module.exports = function (grunt) {
     }
 
     // Default task
-    grunt.registerTask('default', ['clean', 'validate', 'prepare', 'compile', 'test', 'analyse']);
+    grunt.registerTask('default', ['install', 'clean', 'validate', 'compile', 'test', 'analyse']);
```

Comment:
`clean` cleans out the build dirs rather than the deps. i put `install` first incase it updates `grunt-contrib-clean` in some important way. 

was going to add a `jspm clean` to your jspm install task too @piuccio, but thought i'd leave it to, if you want it?